app/src/main/res/layout/welcome.xml

```diff
@@ -43,15 +43,15 @@
         android:layout_height="wrap_content"
         android:fontFamily="sans-serif"
         android:text="@{viewModel.computeProfileNameText()}"
-        android:textColor="@color/oppia_primary_text"
+        android:textColor="@color/component_color_home_activity_greeting_text_color"
         android:textSize="24sp" />
     </com.google.android.flexbox.FlexboxLayout>
 
     <View
       android:layout_width="0dp"
       android:layout_height="6dp"
       android:layout_marginTop="4dp"
-      android:background="@color/color_def_oppia_light_brown"
+      android:background="@color/component_color_home_activity_layout_below_greeting_text_color"
```

Comment:
> This is a background not a text color.

Yes this is a background color but I named it as 'below_greeting_text', if we will name it as background_color then it may create confusion of the background color of home_activity.

Reply:
In that case maybe rename it to `component_color_home_activity_layout_greeting_text_line_color`

Also, merge with latest develop.

Reply:
> In that case maybe rename it to `component_color_home_activity_layout_greeting_text_line_color`
> 
> Also, merge with latest develop.

Done